src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -302,15 +304,36 @@ private boolean canSettle(Authentication a) {
             return true;
         }
         final Set<String> submitters = Sets.newHashSet(submitter.split(","));
-        if (submitters.contains(a.getName()))
+        if (isMemberOf(a.getName(), submitters))
             return true;
         for (GrantedAuthority ga : a.getAuthorities()) {
-            if (submitters.contains(ga.getAuthority()))
+            if (isMemberOf(ga.getAuthority(), submitters))
```

Comment:
Not really a part of this PR, but when would `GrantedAuthority` match the submitter? If a group is used?